### ISSUE TITLE
Adapter port issue

### ISSUE THREAD

**Issue by ShoelessTom** [robertdavidgraham/masscan#756]
`C:\Users\user\masscan\bin>masscan --resume paused.conf --adapter-port 41375
FAIL: source port range must be even power of two: adapter-port=41374-41390
`
How do I fix this?
Even if I set a adapter port between those numbers, it still shows that error msg.

**Comment by NotoriousRebel** [comment]
Ran into the same issue, any luck resolving it?

**Comment by ShoelessTom** [comment]
Yes, my paused.conf included the wrong adapter port, and more things.
Use this tutorial https://github.com/robertdavidgraham/masscan/issues/690
Or simular to change the adaptor port and stuff
